### DISCUSSION TITLE
Deeplinking on iOS

### DISCUSSION THREAD

**Discussion by Lascorbe** [WalletConnect/walletconnect-monorepo#2007] [Developer Support]
1. What should we put on the Mobile Linking's deeplink field (on cloud.walletconnect.com): `example://` or `example://wc?uri=`?
2. What's the best practice to have several deeplinks for different apps which is in fact the same one (e.g. debug, staging, prod versions of the same app, they have a different bundleID and different deeplink), is it ok to create a project for each? should that project be published or can stay as a draft?

**Comment by Lascorbe** [comment]
can someone from the Wallet Connect team reply? Thanks 🙏

**Reply by Batxent** [reply]
seems no-one will reply

**Comment by Lascorbe** [comment]
hi @pedrouid, can you or someone from the team shed some light here? thank you 🙏

**Comment by Lascorbe** [comment]
still waiting a response on this matter :)

**Comment by Arthurpfz** [comment]
maybe we will get an answer by adding tags: https://github.com/orgs/WalletConnect/discussions. _"Remember to tag your discussion post with the relevant environment tag, such as iOS, Android, Web, Flutter, or Unity, to help us better understand your issue and provide tailored support."_
